.github/copilot-instructions.md

```diff
@@ -0,0 +1,193 @@
+# NVIDIA NeMo Curator
+Always reference these instructions first and fallback to search or bash commands only when you encounter unexpected information that does not match the info here.
+
+NVIDIA NeMo Curator is a Python library for fast and scalable data processing and curation for generative AI use cases such as foundation language model pretraining, text-to-image model training, domain-adaptive pretraining, supervised fine-tuning, and parameter-efficient fine-tuning. It leverages GPUs with Ray and RAPIDS for significant performance improvements.
+
+## Working Effectively
+
+### Prerequisites and Environment Setup
+- **Python Version**: 3.12 ONLY. Python 3.11 is NOT supported due to RAPIDS compatibility issues.
```

Comment:
This is not true? Maybe it only thinks this because we don't have explicit tests for 3.11? Or maybe my assumption is wrong?